mypy/checker.py

```diff
@@ -1699,6 +1700,51 @@ def visit_class_def(self, defn: ClassDef) -> None:
         if typ.is_protocol and typ.defn.type_vars:
             self.check_protocol_variance(defn)
 
+    def check_init_subclass(self, defn: ClassDef) -> None:
+        """
+        Check that keywords in a class definition are valid arguments for __init_subclass__().
```

Comment:
Move this to the line above, right after quotes. Essentially we just follow https://www.python.org/dev/peps/pep-0257/